### PR TITLE
Milestone 0.2

### PR DESCRIPTION
**Pull request by casarf**
The communication between Seeeduino and TTN has been tested and configured

### CODE REVIEW

File: sus.txt [hunk 1]
```diff
@@ -0,0 +1,247 @@
+//code for seed-martel-001
```
Comment (chicco785) [inline sus.txt]: what is this file about?

File: diorama/src/waste/examples/UltrasonicDisplayOnTerm/UltrasonicDisplayOnTerm.ino [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/*
```
Comment (chicco785) [inline diorama/src/waste/examples/UltrasonicDisplayOnTerm/UltrasonicDisplayOnTerm.ino]: if these are official files from someone else, let's put references in the README, but let's not store them here.

**Review by chicco785 — COMMENTED** [review]
have a look at my comments

File: sus.txt (hunk 1 above)
Comment (casarf) [reply sus.txt]: The file was a backup for testing purposes, left in the folder by mistake.

File: diorama/src/waste/main_waste_seed_martel_001/main_waste_seed_martel_001.ino [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+    RangeInCentimeters = (int)ultrasonic.MeasureInCentimeters(); // two measurements should keep an interval
+    sprintf(buf2, "%i",RangeInCentimeters);
+    strcat(distanceMessage,buf2);
+    strcat(distanceMessage," cm");
+
+    Serial.println(distanceMessage);
+
+    return distanceMessage;
+}
```
Comment (chicco785) [inline diorama/src/waste/main_waste_seed_martel_001/main_waste_seed_martel_001.ino]: aggiungi la linea a fine codice

File: ev-charging/config.ini [hunk 4]
```diff
@@ -0,0 +1,17 @@
+[connection]
```
Comment (chicco785) [inline ev-charging/config.ini]: sono ancora qui....

**Review by chicco785 — CHANGES_REQUESTED** [review]

File: ev-charging/config.ini (hunk 4 above)
Comment (casarf) [reply ev-charging/config.ini]: Files have been removed

**Review by chicco785 — CHANGES_REQUESTED** [review]
what's the difference between main and cayenne folders? can you document them?

File: README_TTN.md [hunk 5]
```diff
@@ -10,70 +10,84 @@ Official tutorial: https://www.thethingsindustries.com/docs/gateways/thethingski
...
+martel-ttn-01
 
 Account Server:
 https://eu1.cloud.thethings.network/
 
 Gateway Key:
-NNSXS.V2QQSQL73SK5CRMVTPHS24PMIPMXOC6ZIKZS7DY.H7SFIQH2FNHITEJBELFZKKNVPZQRIF5Z22N7LE4GWAA5WX3IZJ2Q
+NNSXS.NACIYY37UADMZG6O6XD7HRH6OM6XDHEKEIUU4CI.VSZHH6U77GJZHBQB2LNO6RQN3RFONSVYEUKI3OBZXPJGCU5EB4KQC0MM3NT01T4L1ANO
 ```
+- Ricordare di rimuovere la parte di sicurezza.
```
Comment (chicco785) [inline README_TTN.md]: english, thx :)

File: diorama/src/waste/note.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+## Required libraries
```
Comment (chicco785) [inline diorama/src/waste/note.md]: move the notes in the README

File: README_TTN.md (hunk 5 above)
Comment (casarf) [reply README_TTN.md]: Giacomo had asked me to write it in Italian, I can modify it if you prefer.

**Review by chicco785 — APPROVED** [review]

**Pull request merged**